Makefile

```diff
@@ -391,3 +391,13 @@ apidiff-build:
 .PHONY: apidiff-compare
 apidiff-compare:
 	@$(foreach pkg,$(ALL_PKGS),$(call exec-command,./internal/buildscripts/compare-apidiff.sh -p $(pkg)))
+
+# Compare API state snapshots in GitHub Actions
+.PHONY: apidiff-compare-GA
```

Comment:
Is it possible to use the same target for both? Perhaps the input dir can have a default value that would work for local runs, and this value would be overridden by GitHub Actions?

Reply:
Currently, apidiff-compare has a default value in the script for input dir which works locally. I created a new target, because I needed to pass the input dir from the workflow. Is there a different way of setting a default value that you were thinking of?

Reply:
There are a couple of options. For the operator, we have something like this:

https://github.com/open-telemetry/opentelemetry-operator/blob/6d76f00573c5a6a00d33d6f8bf2dac6ba107f295/Makefile#L36-L41

You could also use this pattern, where a call to `apidiff-compare-ci` would set an env var and just call the dependent target:

https://github.com/jaegertracing/jaeger-operator/blob/b98361cf4a8fc05e558001b310114aa5e57f8c35/Makefile#L214-L216

Reply:
Updated the workflow to reuse the makefile target, apidiff-compare.